src/patternfly/components/TextInputGroup/text-input-group.hbs

```diff
@@ -1,6 +1,16 @@
-<div class="pf-c-text-input-group{{#if text-input-group--modifier}} {{text-input-group--modifier}}{{/if}}"
+<div class="pf-c-text-input-group
+  {{#if text-input-group--modifier}}
+    {{text-input-group--modifier}}
+  {{/if}}
+  {{#if text-input-group--IsDisabled}}
```

Comment:
```suggestion
  {{~#if text-input-group--IsDisabled}}
```

Reply:
The `~` will remove any leading white-space to keep extra spaces from showing up in the class list. Also, we typically have the `--modifiers` block last, after any other classes we're adding.